server/events/policy_check_project_command_builder.go

```diff
@@ -0,0 +1,89 @@
+package events
+
+import (
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/events/vcs"
+	"github.com/runatlantis/atlantis/server/events/yaml"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+)
+
+func NewPolicyCheckProjectCommandBuilder(p *DefaultProjectCommandBuilder) *PolicyCheckProjectCommandBuilder {
+	return &PolicyCheckProjectCommandBuilder{
+		ProjectCommandBuilder: p,
+		ParserValidator:       p.ParserValidator,
+		ProjectFinder:         p.ProjectFinder,
+		VCSClient:             p.VCSClient,
+		WorkingDir:            p.WorkingDir,
+		WorkingDirLocker:      p.WorkingDirLocker,
+		CommentBuilder:        p.CommentBuilder,
+		GlobalCfg:             p.GlobalCfg,
+	}
+}
+
+type PolicyCheckProjectCommandBuilder struct {
```

Comment:
nit: I would probably have named this PolicyCheckEnabledProjectCommandBuilder. but it's pretty verbose so up to u